project-base/storefront/components/Pages/Order/ContactInformation/FormBlocks/ContactInformationUser.tsx

```diff
@@ -69,6 +46,19 @@ export const ContactInformationUser: FC = () => {
                     }}
                 />
             </FormColumn>
-        </>
+            <TextInputControlled
+                control={formProviderMethods.control}
+                formName={formMeta.formName}
+                name={formMeta.fields.telephone.name}
+                render={(textInput) => <FormLine>{textInput}</FormLine>}
+                textInputProps={{
+                    label: formMeta.fields.telephone.label,
+                    required: true,
+                    type: 'tel',
+                    autoComplete: 'tel',
+                    onChange: (event) => updateContactInformation({ telephone: event.currentTarget.value }),
+                }}
+            />
+        </FormBlockWrapper>
```

Comment:
Why was the order of form elements changed?

Reply:
It was inspired by breno and agreed by Martin